helix-core/src/test/java/org/apache/helix/integration/task/TestEnqueueJobs.java

```diff
@@ -41,13 +47,39 @@
   public void beforeClass() throws Exception {
     setSingleTestEnvironment();
     super.beforeClass();
+    //WorkflowConfig.disableJobPurge();
```

Comment:
What is this?

Reply:
The should be removed. 

Previously, we tried to set `protected static final long DEFAULT_JOB_PURGE_INTERVAL = 30 * 60 * 1000; // default 30 minutes` to -1 in WorkflowConfig.java, by using this function. 

This has its drawback as people have concerns as it pollutes production code as:

1/ need to make DEFAULT_JOB_PURGE_INTERVAL not final.
2/ The WorkflowConfig.disableJobPurge needs to be public. Thus, not testOnly.